test/integration/kubernetes_deploy_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class KubernetesDeployTest < KubernetesDeploy::IntegrationTest
+  def test_basic
+    runner = KubernetesDeploy::Runner.new(
+      namespace: @namespace,
+      current_sha: "abcabcabc",
+      context: "minikube",
```

Comment:
Should we have a helper for the runner as we do with the client to pass the context and namespace automatically? I don't love that minikube is hardcoded in here, the namespace I can live with for sure, as it makes sense for the successive calls like `#get_pods`.